act/Makefile

```diff
@@ -3,39 +3,58 @@
 # Database SID, login credentials are provided in the environment.
 ACCOUNT=$(USERNAME)/$(PASSWORD)@$(SID)
 
+# Note: exit code of sqlplus is unreliable, so we use sql (i.e. sqlcl)
+
 act_shrine_ontology_mapping: status/table_access
 
-# Make a directory for status, i.e. completion times
-# Then as each script finishes, touch a corresponding file.
-status/README:
+
+status/table_access: status/demographics status/visit_details \
+		status/diagnosis status/labs status/procedures
+	sql $(ACCOUNT) @update_table_access.sql $(metadata_schema)
+	touch $@
+
+status/visit_details:
+	sql $(ACCOUNT) @visit_details_map.sql \
+		$(metadata_schema) $(shrine_ont_schema) $(heron_data_schema)
 	mkdir -p status
-	echo "These files represent completion status of SQL scripts." >$@
+	touch $@
 
 status/demographics: status/meta_manual_mapping
-	sqlplus $(ACCOUNT) @demographics_mapping.sql $(metadata_schema) $(shrine_ont_schema)
+	sql $(ACCOUNT) @demographics_mapping.sql $(metadata_schema) $(shrine_ont_schema)
```

Comment:
META_MANUAL_MAPPING.csv is being used in demographics_mapping.sql, so I would request to do that step first.

Reply:
That's what line 22 says, right? `status/demographics` depends on `status/meta_manual_mapping`. `make` ensures that dependencies are satisfied before this stanza is run.